test/transcoding/AllowIntrinsics.ll

```diff
@@ -1,27 +1,52 @@
-; The test checks command-line option for the translator
-; which will allow to represent unknown llvm intrinsics as external function call in SPIR-V.
+; The test checks command-line option for the translator which allows to represent
+; unknown llvm intrinsics (or any custom intrinsics) as external function calls in SPIR-V.
```

Comment:
> (or any custom intrinsics)

nit: I would remove the word "any" since we only allow specific custom intrinsics, not all of them.

Reply:
This is fair for ESIMD use-cases that have facilitated the change, however the Translator change itself may be re-used for other purposes. IMO, this note rather relates to the idea of translating any intrinsics as external function calls, be those LLVM or custom non-LLVM ones, than to how the option filters specific intrinsics out.

That said, I think there's a way to simplify the comment so as to avoid any confusion, so I'll go ahead and employ it :D